src/resources/js/routes/index.js

```diff
@@ -4,15 +4,17 @@ import profile from './profile';
 import search from './search';
 import alert from './alert';
 import message from './message';
+import setting from './setting';
 
 Vue.use(VueRouter);
 
 const routes = [
-  ...profile,
-  ...search,
-  ...alert,
-  ...message
-];
+  ...profile, 
+    ...search, 
+    ...alert, 
+    ...message, 
+    ...setting
+  ];
```

Comment:
[imo]
インデントがちょいズレてる... 👁️ 

```suggestion
const routes = [
  ...profile,
  ...search,
  ...alert,
  ...message,
  ...setting
];
```